plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/object/struct/EditDictionaryPage.java

```diff
@@ -78,8 +81,13 @@ public DBVEntity getDictionary()
     @Override
     protected void createContentsBeforeColumns(Composite panel)
     {
-        Label label = UIUtils.createControlLabel(panel,
-            "Choose dictionary description columns or set custom criteria");
+        Link label = UIUtils.createLink(panel, EditorsMessages.dialog_struct_edit_dictionary_tip, new SelectionAdapter() {
+            @Override
+            public void widgetSelected(SelectionEvent e) {
+                // FIXME: Cannot use constant due to circular dependency.
+                UIUtils.showPreferencesFor(null, null, /*PrefPageDataViewer.PAGE_ID*/ "org.jkiss.dbeaver.preferences.main.dataviewer");
```

Comment:
Not sure what we can do about it.

The only thing is to consider moving `PrefPageDataViewer` along with `TargetPrefPage` to another module.